.evergreen/node-gyp-bug-workaround.sh

```diff
@@ -32,17 +32,20 @@ if [ x"$LOCALAPPDATA" = x"" ]; then
   exit
 fi
 
-set -x
+set -ex
+
+SCRIPTDIR="$(cd $(dirname "$0"); pwd)"
 CACHEDIR="$LOCALAPPDATA/node-gyp/Cache"
 rm -rvf "$CACHEDIR"
 mkdir -p "$CACHEDIR/$NODE_JS_VERSION"
 cd "$CACHEDIR/$NODE_JS_VERSION"
-curl -sSfLO "https://nodejs.org/download/release/v$NODE_JS_VERSION/node-v$NODE_JS_VERSION-headers.tar.gz"
+
+bash "${SCRIPTDIR}/retry-with-backoff.sh" curl -sSfLO "https://nodejs.org/download/release/v$NODE_JS_VERSION/node-v$NODE_JS_VERSION-headers.tar.gz"
```

Comment:
`curl` actually already has a `retry` feature but it seems reliable only in more recent versions that I'm not sure are available everywhere in EVG, just opted for a separate script